data/json/items/gun/45.json

```diff
@@ -84,15 +86,17 @@
     "type": "GUN",
     "name": "L39B-45",
     "description": "After the success of the 9mm model, Leadworks LLC released an L39B chambered in .45.  It soon earned a reputation as one of the most intimidating guns on the market, for firer and target alike.",
-    "ammo": "45"
+    "ammo": "45",
+    "ranged_damage": 1
```

Comment:
Please follow the existing field ordering rather than adding the fields at random. `ranged_damage` should always come after `ammo`